package.json

```diff
@@ -11,7 +11,8 @@
     "scripts": {
         "start": "NODE_ENV=development babel-watch src --extensions \".js,.jsx,.ts,.tsx\"",
         "clean": "rimraf build",
-        "build": "babel src --out-dir build --extensions \".js,.jsx,.ts,.tsx\" && tsc && cp package.json build/",
+        "build-babel": "babel src --out-dir build --extensions \".js,.jsx,.ts,.tsx\"",
```

Comment:
This way I can use ``--watch`` option.

However our current workflow for local development with ``d2-api`` and ``d2-ui-components`` at the same time is overly complex. We should think a better way to join everything, maybe with some advanced webpack module resolution?